python/GafferImageUI/CatalogueUI.py

```diff
@@ -1116,20 +1164,38 @@ def __columnContextMenuDefinition( self ) :
 
 		return menu
 
+	def __outputIndexContextMenuDefinition( self, image ) :
+
+		menu = IECore.MenuDefinition()
+		for i in range( 1, 5 ) :
+
+			menu.append( "/Output Index %i" % i, {
+				"checkBox" : image["outputIndex"].getValue() == i,
+				"command" : functools.partial( Gaffer.WeakMethod( self.__setComparisonImage ), i, image  ),
+			} )
```

Comment:
I feel like this might read a bit better if we omitted the repeated "Output Index" on every line, and instead put that in the menu title? So the first thing you read on each line is the part that makes it unique?